test/nodetool/rest_api_mock.py

```diff
@@ -213,6 +213,7 @@ async def catching_handler(request) -> aiohttp.web.Response:
         aiohttp.web.route("*", "/snitch/{part1}", wrap_handler(server.handle_generic_request)),
         aiohttp.web.route("*", "/storage_proxy/{part1}", wrap_handler(server.handle_generic_request)),
         aiohttp.web.route("*", "/storage_service/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_service/{part1}/", wrap_handler(server.handle_generic_request)),
```

Comment:
yes, possibly a misplaced hunk

Reply:
I checked and no, this hunk was added as part of this patch, because Origin (legacy nodetool) does a request to `/storage_service/ownership/` and the existing `/storage_service/{part1}` didn't cover it unfortunately.